packages/neon_framework/lib/src/widgets/unified_search_results.dart

```diff
@@ -30,20 +30,45 @@ class NeonUnifiedSearchResults extends StatelessWidget {
   Widget build(BuildContext context) {
     final accountsBloc = NeonProvider.of<AccountsBloc>(context);
     final bloc = accountsBloc.activeUnifiedSearchBloc;
-    return ResultBuilder.behaviorSubject(
-      subject: bloc.providers,
-      builder: (context, providers) => NeonListView(
-        scrollKey: 'unified-search',
-        isLoading: providers.isLoading,
-        error: providers.error,
-        onRefresh: bloc.refresh,
-        itemCount: providers.data?.length ?? 0,
-        itemBuilder: (context, index) {
-          final provider = providers.requireData[index];
-
-          return NeonUnifiedSearchProvider(
-            provider: provider,
-            onSelected: onSelected,
+    return StreamBuilder(
+      stream: bloc.isExtendedSearch,
+      builder: (context, isExtendedSearchSnapshot) => ResultBuilder.behaviorSubject(
+        subject: bloc.providers,
+        builder: (context, providers) {
+          final isExtendedSearch = isExtendedSearchSnapshot.data ?? true;
+
+          var itemCount = providers.data?.length ?? 0;
+          if (!isExtendedSearch) {
+            itemCount++;
+          }
+
+          return NeonListView(
+            scrollKey: 'unified-search',
+            isLoading: providers.isLoading,
+            error: providers.error,
+            onRefresh: bloc.refresh,
+            itemCount: itemCount,
+            itemBuilder: (context, index) {
+              if (!isExtendedSearch && index == itemCount - 1) {
+                return Padding(
+                  padding: const EdgeInsets.only(top: 10),
+                  child: Align(
```

Comment:
Wht is the alignment doing?

Reply:
Forcing the button to not take up the full width.